docs/api/en/helpers/BoxHelper.html

```diff
@@ -13,8 +13,8 @@
 		<h1>[name]</h1>
 
 		<p class="desc">
-			Helper object to show the world-axis-aligned bounding box around an object.
-
+			Helper object to show the world-axis-aligned bounding box around an object. 
+			It can be used in synergy with [page:Box3] items in order to easily compute the [link:https://en.wikipedia.org/wiki/Minimum_bounding_box Minimum Bounding Boxes] for objects, and use them for fast collision detection. See [page:Box3.intersect].
```

Comment:
I feel this is misleading. You don't need a BoxHelper object to create bounding boxes for objects and I don't think it should be implied that the box helper should be used for anything other than debugging.

Reply:
You don't need it, but AFAIK using a BoxHelper is the only way to automatically resize the bounding box every time the object changes position, rotation or scale. I'd argue its visual use for debug is actually secondary in relevance compared to what the average developer can get from it as an easy way to keep your bounding boxes consistent. If there's another way I'm not aware of I'll be glad to learn about it. I can rephrase it to say it's used to compute an axis-aligned bounding box though.

Reply:
While you don't really need it, it would be nice to at least give some hint that you can also calculate the bounding box this way while debugging. Also, I just edited the description to point out that it's the axis-aligned bounding box to be calculated and not the minimum one, but if that is true shouldn't the Box3 page of the docs also be corrected? I can take care of it as well if that's the case.

> Represents a box or cube in 3D space. The main purpose of this is to represent the Minimum Bounding Boxes for objects. 
https://threejs.org/docs/#api/en/math/Box3

Reply:
I think the docs should captured the designed intent of the class, which is just for debugging -- I don't think it's a good idea to encourage this kind of use of BoxHelper in the docs. The BoxHelper inherits from `Object3D` so while I'm not sure if it makes sense to do it implies that this will work:

```js
var box3 = new THREE.Box3();
box3.setFromObject( helper );
```

But for any production use the box3 object should just call `setFromObject` on the original target.

> if that is true isn't the Box3 page of the docs also incorrect?

I would think so. To me a "minimum bounding box" needs to be able to be oriented to minimize the volume of the box. "Minimum axis-aligned bounding box" might be better.

Reply:
> I think the docs should captured the designed intent of the class, which is just for debugging -- I don't think it's a good idea to encourage this kind of use of BoxHelper in the docs.
Got it, I'll try to make it clearer in the description of the class/

> The BoxHelper inherits from `Object3D` so while I'm not sure if it makes sense to do it implies that this will work:
> 
> ```js
> var box3 = new THREE.Box3();
> box3.setFromObject( helper );
> ```
> 
> But for any production use the box3 object should just call `setFromObject` on the original target.

I agree, but wouldn't it be better to at least point out in the example how you can use the `update()` method? I can slightly edit the example so that it both shows the correct way of updating the Box3 in production and the way you update the helper. 

> > if that is true isn't the Box3 page of the docs also incorrect?
> 
> I would think so. To me a "minimum bounding box" needs to be able to be oriented to minimize the volume of the box. "Minimum axis-aligned bounding box" might be better.

Gonna correct that as well then.